### DISCUSSION TITLE
QRave Stretched symbology rasters don't transfer from Arc

### DISCUSSION THREAD

**Discussion by MattReimer** [Riverscapes/RiverscapesXML#310] [RAVE: Symbology/BusinessLogic] [answered]
This message was originally from @CHafen 

Hey @Riverscapes/riverscape-project-curators 
I noticed that rasters with stretched symbology and no set high and low values, don't transfer over to QRAVE very well, ex
![Arc_MeanPrecip](https://user-images.githubusercontent.com/39837888/138349285-fd605b70-becc-4468-9e17-d9fb333dfc6a.png)
![Arc_MeanPrecip2](https://user-images.githubusercontent.com/39837888/138349297-9474b5fc-73b8-421e-8d4f-8e46747cb1ce.png)
.
These look great in Arc, but Q. Looks like this
![Q_MeanPrecip](https://user-images.githubusercontent.com/39837888/138349330-8aad6852-99af-4768-9927-9d0f1805f657.png)
.
![Q_MeanPrecip2](https://user-images.githubusercontent.com/39837888/138349891-e6b98bc3-a7fe-440d-b263-4ee4dfd47838.png)
.
The other climate rasters look great, but have a set high and low that I believe stays the same for all projects.

I've been working on the symbology for GUT QRAVE an
![Arc_Slope](https://user-images.githubusercontent.com/39837888/138349679-e38595ec-07fb-45dd-aaa8-937ddf7bab5f.png)
![Q_Slope](https://user-images.githubusercontent.com/39837888/138349691-a92a1c4f-800f-48d3-b453-ef09de50b524.png)

I have had the same issue as I've tried to create qml for raster files

Any Ideas how to fix this?

> Also, hillshade and DEM don't have this problem. Not sure why they work and others don't...

**Comment by CHafen** [comment]
@MattReimer I was just looking over Riverscapes Context to get familiar with it and noticed this. The projects are <https://northarrowresearchlabs.github.io/riverscapes-staging//#/Anabranch/5677bb79-2d8f-4463-acff-3e3510657feb> and <https://northarrowresearchlabs.github.io/riverscapes-staging//#/Anabranch/bc5d43bd-3cde-4dbe-9973-4eb7d836812d>

**Reply by MattReimer** [reply]
This is a really good find. Qml files can get locked to specific values and if that doesn't autoscale properly then we need to figure this out. 

Before I make a Qrave bug about it I want to explore a bit and see what the edges of this problem are. Thank you for posting the links

**Reply by CHafen** [reply]
Do these links work for you @MattReimer? When I try to click on them it says page not found.
They Riverscapes Context 05020004 and 08010100

**Reply by MattReimer** [reply]
Oh. I see the problem. In your markdown you had `[https://nor....](url)` so it thought the link was actually `http://url`

You can use angle brackets when the link and the text are the same like this `<https://north...>`

**Reply by CHafen** [reply]
The GUT symbology isn't up yet as I'm still finishing it up, so all the qmls for that are still just on my computer.

**Comment by joewheaton** [comment, marked as answer]
@MattReimer provided answer:

> Oh. I see the problem. In your markdown you had [https://nor....](url) so it thought the link was actually http://url
> 
> You can use angle brackets when the link and the text are the same like this <https://north...>
